pyproject.toml

```diff
@@ -26,12 +26,12 @@ gradio = "^3.1.7"
 colorama = ">=0.4.5,<1.0.0"
 jiwer = "^2.5.1"
 transformers = "^4.23.0"
-soundfile = ">=0.11.0,<1.0.0"
+soundfile = "^0.11.0"
 evaluate = ">=0.3.0,<1.0.0"
 pyctcdecode = ">=0.4.0,<1.0.0"
-librosa = ">=0.9.2,<1.0.0"
+librosa = "0.9.2"
```

Comment:
Hesitant to include a specific version. Wouldn't 0.9.1 or 0.9.3 work as well? What doesn't work?